app/utils/nonsipp/TaskListStatusUtils.scala

```diff
@@ -401,6 +401,34 @@ object TaskListStatusUtils {
         }
     }
 
+  private def getDisposalIncompleteIndex[A, B](
```

Comment:
This now becomes unused, but IMO there is a slight chance we're going to use in some other disposal journey.

Reply:
happy to leave it in for now